### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by mcarpenterjr** [Glavin001/atom-beautify#922]
### Description

atom-beautify would not update to v0.29.2, uninstalled and attempted to reinstall.

### Expected Results

atom-beautify should have installed but fails with this error:

```

> spawn-sync@1.0.15 postinstall C:\Users\MARKNC~1\AppData\Local\Temp\apm-install-dir-116314-4772-at3zgv\node_modules\atom-beautify\node_modules\cross-spawn\node_modules\spawn-sync
> node postinstall

Installing native dependencies (this may take up to a minute)

npm WARN optional dep failed, continuing fsevents@1.0.11
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN optional dep failed, continuing source-map@0.1.43
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm WARN engine is-buffer@1.1.3: wanted: {"node":">=0.12"} (current: {"node":"0.10.40","npm":"2.13.3"})
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\Mark N Carpenter Jr\\AppData\\Local\\atom\\app-1.7.0\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\Mark N Carpenter Jr\\AppData\\Local\\atom\\app-1.7.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\Mark N Carpenter Jr\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\Mark N Carpenter Jr\\.atom\\.apmrc" "install" "C:\\Users\\MARKNC~1\\AppData\\Local\\Temp\\d-116314-4772-105dk6o\\package.tgz" "--target=0.36.8" "--arch=ia32"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! file C:\Users\Mark N Carpenter Jr\.atom\.apm\amdefine\1.0.0\package\package.json
npm ERR! code EJSONPARSE

npm ERR! Failed to parse json
npm ERR! Unexpected token '\u0000' at 1:1
npm ERR! 
npm ERR! ^
npm ERR! File: C:\Users\Mark N Carpenter Jr\.atom\.apm\amdefine\1.0.0\package\package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR! 
npm ERR! This is not a bug in npm.
npm ERR! Tell the package author to fix their package.json file. JSON.parse

```

**Comment by Glavin001** [comment]
Duplicate of many previous related issues: https://github.com/Glavin001/atom-beautify/search?q=cross-spawn&type=Issues&utf8=%E2%9C%93

**Comment by Glavin001** [comment]
I just published a patch in release v0.29.4 that makes `cross-spawn` dependency optional. 
The [`cross-spawn`](https://github.com/IndigoUnited/node-cross-spawn) package depends on [`spawn-sync`](https://github.com/ForbesLindesay/spawn-sync) and has been causing problems when installing Atom-Beautify for many users. By making this dependency optional, users will be able to easily install Atom-Beautify and, in the case that [`cross-spawn`](https://github.com/IndigoUnited/node-cross-spawn) fails to install, will fallback to the builtin [`spawn`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) method.
